garak/__init__.py

```diff
@@ -3,3 +3,23 @@
 __version__ = "0.10.3.post1"
 __app__ = "garak"
 __description__ = "LLM vulnerability scanner"
+
+import logging
+import os
+from garak import _config
+
+GARAK_LOG_FILE_VAR = "GARAK_LOG_FILE"
+
+# allow for a file path configuration from the ENV and set for child processes
+_log_filename = os.getenv(GARAK_LOG_FILE_VAR, default=None)
+if _log_filename is None:
+    _log_filename = _config.transient.data_dir / "garak.log"
+    os.environ[GARAK_LOG_FILE_VAR] = str(_log_filename)
```

Comment:
I like this. We have to be able to specify a logfile path and we also have to be able to log what goes wrong during configuration, which includes setting a logfile path.